packages/components/src/components/number-input/_number-input.scss

```diff
@@ -111,7 +111,7 @@
     justify-content: center;
     align-items: center;
     width: rem(32px);
-    height: rem(20px);
+    height: calc(#{$input-height} / 2);
```

Comment:
Any reason we are using calc here, but on L232 we are doing an inline calc?

Reply:
Good catch @tw15egan! I meant to use `calc()` for other places, too, because Sass-based calculation does not work if e.g. `$input-height` has a CSS custom property as the value.